logtail/buffer.go

```diff
@@ -62,7 +62,7 @@ func (m *memBuffer) Write(b []byte) (int, error) {
 	defer m.dropMu.Unlock()
```

Comment:
what's this mutex doing? It implies it guards dropCount but TryReadLine checks dropCount without this mutex?

Reply:
`TryReadLine` apparently checks a different field also called `dropCount`, so there's no race here.